pkg/plugins/golang/options.go

```diff
@@ -64,17 +54,7 @@ var (
 
 // Options contains the information required to build a new resource.Resource.
 type Options struct {
-	// Group is the resource's group. Does not contain the domain.
-	Group string
-	// Domain is the resource's domain.
-	Domain string
-	// Version is the resource's version.
-	Version string
-	// Kind is the resource's kind.
-	Kind string
-
```

Comment:
These changes are breaking changes for the lib. Why do we need to change the options to address this need?

Reply:
These flags need to be handled by the cli because every plugin that creates an api will need the GVK, thats why the flags need to be binded by the cli and not by a certain plugin or all plugins will be incompatible as they will all have these flags.